scripts/image-builder/build.sh

```diff
@@ -173,7 +191,7 @@ while [ $# -gt 0 ] ; do
         shift
         LVM_SYSROOT_SIZE="$1"
         [ -z "${LVM_SYSROOT_SIZE}" ] && usage "System root LVM partition size not specified"
-        [ ${LVM_SYSROOT_SIZE} -lt ${LVM_SYSROOT_SIZE_MIN} ] && usage "System root LVM partition size cannot be smaller than ${LVM_SYSROOT_SIZE_MIN}MB"
+        [ "${LVM_SYSROOT_SIZE}" -lt ${LVM_SYSROOT_SIZE_MIN} ] && usage "System root LVM partition size cannot be smaller than ${LVM_SYSROOT_SIZE_MIN}MB"
```

Comment:
`${LVM_SYSROOT_SIZE_MIN}` is unquoted. Would quotes interfere with `-lt` number operator?

Reply:
Bash seems to implement this as expected. since the `-lt` seems to be doing the correct comparison. Verified this locally with the script and some quick tests as well on the terminal. Even some docs use the quote variant as an example, so we should be fairly confident on this behaving as expected. https://tldp.org/LDP/abs/html/comparison-ops.html

```
➜  [ "012" -lt 500 ] && echo less || echo more
less
➜  [ "120" -lt 500 ] && echo less || echo more
less
➜  [ "015" -lt 500 ] && echo less || echo more
less
➜  [ "510" -lt 500 ] && echo less || echo more
more
```